omnigibson/prims/entity_prim.py

```diff
@@ -100,8 +99,33 @@ def _load(self):
         raise NotImplementedError("By default, an entity prim cannot be created from scratch.")
 
     def _post_load(self):
+        # If this is a cloth, delete the root link and replace it with the single nested mesh
+        if self._prim_type == PrimType.CLOTH:
+            # Verify only a single link and a single mesh exists
+            old_link_prim = None
+            cloth_mesh_prim = None
+            for prim in self._prim.GetChildren():
+                if prim.GetPrimTypeInfo().GetTypeName() == "Xform":
+                    assert old_link_prim is None, "Found multiple XForm links for a Cloth entity prim! Expected: 1"
+                    old_link_prim = prim
+                    for child in prim.GetChildren():
+                        if child.GetPrimTypeInfo().GetTypeName() == "Mesh" and not child.HasAPI(UsdPhysics.CollisionAPI):
+                            assert cloth_mesh_prim is None, "Found multiple meshes for a Cloth entity prim! Expected: 1"
+                            cloth_mesh_prim = child
+
+            # Move mesh prim one level up via copy, then delete the original link
+            # NOTE: We copy because we cannot directly move the prim because it is ancestral
+            # NOTE: We use this specific delete method because alternative methods (eg: "delete_prim") fail beacuse
+            # the prim is ancestral. Note that because it is non-destructive, the original link prim path is still
+            # tracked by omni, so we have to have to utilize a new unique prim path for the copied cloth mesh
+            # See omni.kit.context_menu module for reference
+            new_path = f"{self._prim_path}/{old_link_prim.GetName()}_cloth"
+            omni.kit.commands.execute("CopyPrim", path_from=cloth_mesh_prim.GetPath(), path_to=new_path)
+            omni.kit.commands.execute("DeletePrims", paths=[old_link_prim.GetPath()], destructive=False)
```

Comment:
hmm this whole process is not very clean. Is there a away to solve this ancestral issue? Can we do the following?
```
omni.kit.commands.execute("CopyPrim", path_from=cloth_mesh_prim.GetPath(), path_to="temp_prim_path")
for child in old_link_prim.GetChildren():
    omni.kit.commands.execute("DeletePrim", path=child.GetPath())
omni.kit.commands.execute("DeletePrim", path=old_link_prim.GetPath())
omni.kit.commands.execute("MovePrim", path_from="temp_prim_path", path_to=f"{self._prim_path}/base_link")
```